### DISCUSSION TITLE
unnecessary folder and file

### DISCUSSION THREAD

**Discussion by bboydflo** [timlrx/tailwind-nextjs-starter-blog#89] [Q&A] [answered]
Hi, 

I am still studying this starter template as I want to build mine based on it. Inside `pages` folder there is a `blog/page/[page].js` file. It seems that if I remove the entire folder `pages/blog/page` everything still works as expected.

Is that just a remaining thing? Or is it actually being used anywhere?

@timlrx any idea?

**Comment by timlrx** [comment, marked as answer]
It's for the paginated routes e.g. `blog/2` which you will notice if you create more than 10 post or reduce the max post per page variable to a smaller number

**Reply by bboydflo** [reply]
thanks! it makes more sense now after I tested it.
